docs/api/v3.rst

```diff
@@ -175,6 +186,47 @@ Project details
     :statuscode 404: There is no ``Project`` with this slug
 
 
+Project create
+++++++++++++++
+
+.. http:post:: /api/v3/projects/
+
+    Import a project into Read the Docs.
+
+    **Example request**:
+
+    .. sourcecode:: bash
+
+        $ curl \
+          -X POST \
+          -H "Authorization: Token <token>" https://readthedocs.org/api/v3/projects/ \
+          -H "Content-Type: application/json" \
+          -d @body.json
```

Comment:
Should we also include a Python example?

Reply:
That would be pretty handy. I will create an issue for this to use tabs (Python + Slumber and cURL).

Reply:
Issue at https://github.com/readthedocs/readthedocs.org/issues/5935